doc/cygnus-ngsi/flume_extensions_catalogue/orion_ckan_sink.md

```diff
@@ -354,3 +358,44 @@ In detail, this is the workflow when `NGSICKANSink` is combined with `CKANCache`
 
 [Top](#top)
 
+##<a name="section4"></a>Annexes
+
+###<a name="section1"></a>Provisioning a CKAN resource for the column mode
```

Comment:
Sorry, I forgot about this: It must be referenced the CKAN API: http://docs.ckan.org/en/latest/api/

Something like "This section is built upon the assumption you are familiar with the CKAN API. If not, please have a look on [it](http://docs.ckan.org/en/latest/api/)".